Lib/test/test_generators.py

```diff
@@ -1921,7 +1921,7 @@ def printsolution(self, x):
 >>> def f(): (yield bar) += y
 Traceback (most recent call last):
   ...
-SyntaxError: cannot assign to yield expression
+SyntaxError: yield expression is an illegal expression for augmented assignment
```

Comment:
Whoops, you added the quotes here. Sorry!
```suggestion
SyntaxError: 'yield expression' is an illegal expression for augmented assignment
```